scripts/serverless.js

```diff
@@ -419,16 +450,17 @@ const processSpanPromise = (async () => {
 
       hasTelemetryBeenReported = true;
       if (!isTelemetryDisabled) {
-        storeTelemetryLocally(
-          generateTelemetryPayload({
+        storeTelemetryLocally({
+          ...generateTelemetryPayload({
             command,
             options,
             commandSchema,
             serviceDir,
             configuration: configurationFromInteractive,
             commandUsage,
-          })
-        );
+          }),
+          outcome: 'success',
```

Comment:
Ouch! I guess it explains missing `outcome` on some events, as once reported by @mnapoli